src/Client.php

```diff
@@ -513,6 +519,42 @@ public function getWebDriver(): WebDriver
         return $this->webDriver;
     }
 
+    /**
+     * @param UserInterface $user
+     *
+     * @return $this
+     */
+    public function logInUser(
+        object $user,
+        ?SessionStorageInterface $sessionStorage = null,
+        string $firewallContext = 'main',
+    ): self {
+        if (!interface_exists(UserInterface::class)) {
+            throw new \LogicException(sprintf('"%s" requires symfony/security-core to be installed.', __METHOD__));
+        }
+
+        if (!$user instanceof UserInterface) {
+            throw new \LogicException(sprintf('The first argument of "%s" must be instance of "%s", "%s" provided.', __METHOD__, UserInterface::class, get_debug_type($user)));
+        }
+
+        // Make a single request to avoid 'Invalid cookie domain' error when attempting to
+        // set a cookie to the uninitialised web driver. The path does not matter for this
+        $this->request('GET', '/');
+
+        if (null === $sessionStorage) {
+            $sessionStorage = new MockFileSessionStorage('/app/var/cache/panther/sessions');
```

Comment:
This should something like `tempnam(sys_get_temp_dir(), 'panther-sessions');` instead. This hardcoded directory may not be writable.